server/plugin/webhook.go

```diff
@@ -380,6 +386,8 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 		p.API.LogWarn("Failed to render template", "error", err.Error())
 		return
 	}
+	renderedMessage = p.sanitizeDescription(renderedMessage)
```

Comment:
What about PRs?

Reply:
We have done it for both opened issue and Open PR

Reply:
now it looks like this 

![Screenshot 2021-08-19 at 5 57 12 PM](https://user-images.githubusercontent.com/86650698/130068358-1e962bd5-e358-4708-abb7-924626fc84f3.png)

Reply:
It seems like the markdown also get erased. Can you please take a look what is happening there?

Reply:
@hanzei  tried to figure out but i didn't find any difference  tried the same desc with the code in master to compare 
it is same

Reply:
That's surprising. Could you also please test the latest release? On the community server markdown renders fine:

Reply:
@hanzei 
i updated my code for mm-server , mm-webapp , mm-plugin-github
and created a PR and the Message looks like this

![Screenshot 2021-08-20 at 9 45 00 AM](https://user-images.githubusercontent.com/86650698/130178409-c8e1d4fc-e3c0-471a-9715-0c4c84f509f6.png)

![Screenshot 2021-08-20 at 9 44 39 AM](https://user-images.githubusercontent.com/86650698/130178417-8e289d76-02c5-4abd-9e74-42ca5ec851e6.png)

Reply:
What is the content of the original GitHub message? Does it contain md or html?

Reply:
@hanzei  i tried with this
```
Bumps <a href="https://github.com/npm/hosted-git-info">hosted-git-info</a> from 2.8.8 to 2.8.9. <strong>This update includes a security fix.</strong>
```

Reply:
Okay, that's no md input. I that case this PR didn't introduce a regression. LGTM :+1: